src/stateless/icon.vue

```diff
@@ -77,6 +77,10 @@
             <path class="path" d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm4 9H9v3H7V9H4V7h3V4h2v3h3v2z" fill-rule="nonzero"/>
         </svg>
 
+        <svg v-if="name === 'plus-v2'" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we find a better name than `-v2`? It's not very descriptive. Maybe the original one can be `plus-round` and this one just `plus`?

Reply:
I'd rather not rename old ones, maybe ```cross```?

Reply:
Yes, maybe it's safer not to do that now.
What about `plus-square`?

Reply:
👍